helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -822,6 +838,19 @@ public void setGlobalRebalancePreference(Map<GlobalRebalancePreferenceKey, Integ
     return DEFAULT_GLOBAL_REBALANCE_PREFERENCE;
   }
 
+  /**
+   * Set the asynchronous global rebalance mode.
```

Comment:
nit: the method will be used mostly by client library, could you make the java doc more concrete? How often the async rebalance will be triggered, estimated impact, default value and so on?

Reply:
This part has been documented in the GLOBAL_REBALANCE_ASYNC_MODE enum items. I think that would be a better place for detail.